src/backend/utils/compatibility_layers.ts

```diff
@@ -474,3 +481,27 @@ export function getWineFlagsArray(
   }
   return commandArray
 }
+
+export function getUmuPath() {
+  //TODO: figure out how to use searchForExecutableOnPath here
+
+  const paths = ['/app/share', '/usr/local/share', '/usr/share', '/opt']
```

Comment:
In the next release of umu-launcher, these values need to change because the system package is now distributed as a Python module. For this case, just search PATH for `umu-run` instead of `umu_run.py`:

`const paths = ['/app/bin', '/usr/local/bin', '/usr/bin']`